### PR TITLE
[BH-1191] Alarm operations priorities

### PR DESCRIPTION
**Pull request by swichu91**
Implemented a very basic alarm operations priority.

[BH-1204] Fixed time format being incorrectly displayed
in background sounds app.

[BH-1204]: https://appnroll.atlassian.net/browse/BH-1204?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mjan-dev — APPROVED** [review]
LGTM

**Review by wojtekrzepecki — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsProgressPresenter.cpp [hunk 1]
```diff
@@ -92,4 +92,8 @@ namespace app::bgSounds
     {
         return player.isPaused();
     }
+    void BGSoundsProgressPresenter::onBeforeShow()
+    {
+        getView()->setTimeFormat(timeModel->getTimeFormat());
```
Comment (paweljonskim) [inline products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsProgressPresenter.cpp:97]: is this change related to this PR? What does it cause?

**Review by paweljonskim — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsProgressPresenter.cpp (hunk 1 above)
Comment (swichu91) [reply products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsProgressPresenter.cpp:97]: Actually, this PR contains BH-1191 and BH-1204. These changes are related to BH-1204.

Comment (swichu91) [reply products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsProgressPresenter.cpp:97]: It fixes incorrect 24/12 time format being displayed in bg sounds app :)
